Content.Shared/Humanoid/SkinColor.cs

```diff
@@ -17,6 +17,14 @@ public static class SkinColor
     public const float MinFeathersValue = 36f / 100;
     public const float MaxFeathersValue = 55f / 100;
 
+    // Einstein Engines - Tajaran
+    public const float MinAnimalFurHue = 20f / 360;
+    public const float MaxAnimalFurHue = 60f / 360;
```

Comment:
Might wanna change this to incorporate more colors. 20°-60° includes only colors between orange and yellow.